### DISCUSSION TITLE
DOM信息整理插件

### DISCUSSION THREAD

**Discussion by king-bcolor** [NonceGeek/indiehacker-handbook#16] [IdeaStack]
- 灵感来源
   - 在浏览 aave 的 switch 列表时，想快速了解下拉选择器中其他代币的信息，之后我打开 F12  获取复制 html 内容最后让克劳德反馈信息，这一系列操作略显繁琐，想通过浏览器插件的方式简化流程，并在后期拓展更多信息提取等 “```web to AI```” 的快捷功能改善网页体验
   
- 应用场景
  - 适用于快速分析整理网页内容，对于学习者和内容生产者，有利于突破 “``` 请关注公众号（登陆账号）复制内容 ```” 的内容藩篱获取结构化数据用于个体生产与学习

**Comment by leeduckgo** [comment]
> * 灵感来源
>   
>   * 在浏览 aave 的 switch 列表时，想快速了解下拉选择器中其他代币的信息，之后我打开 F12  获取复制 html 内容最后让克劳德反馈信息，这一系列操作略显繁琐，想通过浏览器插件的方式简化流程，并在后期拓展更多信息提取等 “`web to AI`” 的快捷功能改善网页体验
> * 应用场景
>   
>   * 适用于快速分析整理网页内容，对于学习者和内容生产者，有利于突破 “`请关注公众号（登陆账号）复制内容`” 的内容藩篱获取结构化数据用于个体生产与学习

很好👍🏻这个可以和 Web3 相互结合。

**Comment by king-bcolor** [comment]
# DOM 插件 MVP 必要功能点
1. 按住快捷键点击鼠标获取 DOM 内容，效果如同 F12 检查网页元素
2. 获取 html 内容转发给 AI 处理并储存到浏览器本地
3. 展示结果到浏览器侧边栏
4. 收款码代替收款功能

**Comment by king-bcolor** [comment]
repo: https://github.com/king-bcolor/cool-assistant
